src/shared/crypto/bitcoin/p2wsh-p2sh-address-gen.spec.ts

```diff
@@ -6,9 +6,9 @@ import { base58check } from '@scure/base';
 import { HDKey } from '@scure/bip32';
 import { hashP2WPKH } from '@stacks/transactions';
 import { BIP32Factory } from 'bip32';
-import * as bip39 from 'bip39';
 import * as bitcoin from 'bitcoinjs-lib';
 
+import { mnemonicToSeedSync } from './bip39/bip39.utils';
```

Comment:
I'm not sure if these tests are used? Or if they don't run now since https://github.com/leather-wallet/extension/pull/4284 

I tried running and `yarn test` no longer works. 
```
error Command "test" not found. Did you mean "jest"?
```

Running with `yarn jest` and the tests exit and fail. All 46 tests fail

Reply:
We only have `yarn test:unit` and `yarn test:integration` now. #4284 removed the old intgration tests only under tests-legacy.

These are units tests ran with vitest.

```
 DEV  v0.34.6 /Users/kyranjamie/dev/leather/wallet

stdout | unknown test
taprootAddress bc1ppv609nr0vr25u07u95waq5lucwfm6tde4nydujnu8npg4q75mr5sxq8lt3

 ✓ src/shared/crypto/bitcoin/p2tr-address-gen.spec.ts (6)
 ✓ src/shared/crypto/stacks/stacks-address-gen.spec.ts (1)
 ✓ src/shared/crypto/bitcoin/bip322/sign-message-bip322.spec.ts (7)
 ✓ src/app/routes/account-gate.spec.ts (3)
 ✓ src/app/common/transactions/stacks/generate-unsigned-txs.spec.ts (1)
 ✓ src/shared/crypto/bitcoin/p2wsh-p2sh-address-gen.spec.ts (46)
 ✓ src/app/common/transactions/stacks/post-condition.utils.spec.ts (4)
 ✓ src/shared/utils/analytics.spec.ts (6)
 ✓ src/app/common/stacks-utils.spec.ts (4)
 ✓ src/app/common/transactions/bitcoin/coinselect/local-coin-selection.spec.ts (9)
 ✓ src/app/features/activity-list/components/transaction-list/transaction-list.utils.spec.ts (4)
 ✓ src/shared/utils/validate-urls.spec.ts (2)
 ✓ src/app/common/utils/spam-filter.spec.ts (3)
 ✓ src/app/common/date-utils.spec.ts (1)
 ✓ src/app/common/money/calculate-money.spec.ts (9)
 ✓ src/app/store/transactions/post-conditions.spec.ts (1)
 ✓ src/app/store/utils/vault-reducer-migration.spec.ts (4)
 ✓ src/shared/route-urls.spec.ts (1)
 ✓ src/app/common/account-restoration/account-restore.spec.ts (2)
 ✓ src/app/common/utils.spec.ts (5)
 ✓ src/app/common/math/calculate-averages.spec.ts (2)
 ✓ src/app/common/math/helpers.spec.ts (3)
 ✓ src/app/common/validation/validate-tx-id.spec.ts (6)
 ✓ src/shared/utils/requests.spec.ts (1)
 ✓ src/app/common/crypto-assets/stacks-crypto-asset.utils.spec.ts (5)
 ✓ src/shared/crypto/bitcoin/bitcoin.utils.spec.ts (2)
 ✓ src/app/common/transactions/stacks/transaction.utils.spec.ts (2)
 ✓ src/shared/crypto/bitcoin/p2wpkh-address-gen.spec.ts (3)
 ✓ src/app/features/psbt-signer/hooks/find-outputs-receiving-inscriptions.spec.ts (2)
 ✓ src/app/query/bitcoin/address/tests/transactions-by-address.spec.ts (2)
 ✓ src/shared/crypto/bitcoin/bip322/bip322-utils.spec.ts (2)
 ✓ src/app/pages/send/ordinal-inscription/coinselect/select-inscription-coins.spec.ts (2)
 ✓ src/app/query/bitcoin/ordinals/brc20/brc-20.utils.spec.ts (1)
 ✓ src/app/common/string-utils.spec.ts (1)

 Test Files  34 passed (34)
      Tests  153 passed (153)
   Start at  12:20:25
   Duration  3.07s (transform 390ms, setup 7.74s, collect 3.75s, tests 594ms, environment 3ms, prepare 1.58s)

```

Reply:
Thanks 👍 I added a note to the `README`